src/core/wee-config.c

```diff
@@ -3402,6 +3421,22 @@ config_weechat_init_options ()
         &config_check_separator, NULL, NULL,
         &config_change_buffers, NULL, NULL,
         NULL, NULL, NULL);
+    config_look_status_more_prefix = config_file_new_option (
+        weechat_config_file, ptr_section,
+        "status_more_prefix", "string",
+        N_("text to display before count for buffer with new data (status bar)"),
+        NULL, 0, 0, _("-MORE("), NULL, 0,
```

Comment:
For me it's a problem: the default value of an option should not depend on the user locale: any default config file on any system must be the same.

The translated strings should not be customizable themselves, instead we should offer another way to customize this bar item if needed : I see for example two solutions:

1. add another bar item with the more value, without label
2. add options for bar items, so the "scroll" bar item could have an option to disable the label.

And by the way, a script can easily add such bar item.